package.json

```diff
@@ -4,66 +4,63 @@
   "main": "index.js",
   "license": "MIT",
   "scripts": {
-    "build:dev": "webpack",
+    "build:dev": "webpack --mode development",
     "build:prod": "webpack -p --env production",
-    "dev-server": "webpack-dev-server",
+    "dev-server": "webpack-dev-server --open --mode development",
```

Comment:
As I remember, we decided yesterday to use `env` instead of `mode` because you'll need that in your webpack configuration.